package.json

```diff
@@ -70,7 +70,7 @@
     "gulp-sourcemaps": "^1.2.4",
     "gulp-uglify": "^1.0.1",
     "gulp-webserver": "^0.9.1",
-    "jest-cli": "0.4.19",
+    "jest-cli": "0.6.0",
```

Comment:
I was getting errors running `npm test`, after some digging ended up bumping the version which resolved those. However there are still some `warnings` in tests which i think can be take care later in another PR.

Reference
https://github.com/facebook/jest/issues/340
https://github.com/facebook/jest/issues/341